mmv1/templates/terraform.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Base terraform.yaml for product folders that have been converted over to the 
+# per-resource format.
+--- !ruby/object:Provider::Terraform::Config
+# This is for copying files over
+files: !ruby/object:Provider::Config::Files
+  # These files have templating (ERB) code that will be run.
+  # This is usually to add licensing info, autogeneration notices, etc.
+  compile:
+<%= lines(indent(compile('provider/terraform/product~compile.yaml'), 4)) -%>
```

Comment:
note: Every time I remember this part of the file exists, I'm terrified of MMv1's inner workings haha.

Reply:
Trying to remove `terraform.yaml` as a whole (and these lines) is what stumped me at the beginning. Letting it exist somewhere else made it much easier...